src/main/web/app/components/Calendar.js

```diff
@@ -16,7 +16,7 @@ export default function Calendar(props) {
   return (
     <container>
       <div className="calendar">
-        <TimeView start={8} end={18} />
+        <TimeView start={7} end={18} />
```

Comment:
@MalteBartels Is it correct, that we start from 7am now?

Reply:
No it's not and I changed it. I don't know why it appears here